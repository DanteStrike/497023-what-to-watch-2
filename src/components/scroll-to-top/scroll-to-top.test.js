import React from "react";
import renderer from "react-test-renderer";

import {ScrollToTop} from "./scroll-to-top.jsx";


it(`Render correctly ScrollToTop component`, () => {
  const locationMock = {
    pathname: `any`
  };
  const component = renderer
    .create(
        <ScrollToTop
          location={locationMock}
        />
    ).toJSON();

  expect(component).toMatchSnapshot();
});
