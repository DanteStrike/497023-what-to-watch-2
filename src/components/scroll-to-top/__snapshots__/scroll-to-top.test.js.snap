// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly ScrollToTop component 1`] = `null`;
